### ISSUE TITLE
Samsung Selinux Status removed even when enforcing 

### ISSUE THREAD

**Issue by Fede2782** [chiteroman/PlayIntegrityFix#475]
In Samsung devices with Selinux enforcing this build prop must be set to 1 in order to have settings and other apps show the correct selinux status.

@salvogiangri already did a patch to fix this. If the value is already set to 1 there is not need to do anything.

**Comment by essence25** [comment]
Im experiencing this now with PlayIntegrityFix v18.0, TB Checker "SE linux Flag is Enabled = Fail" where it always passed. 
Also in DevInfo "SELinux = Unable to Determine" app has root permission. 
Kernel SU shows enforced and command line "getenforce" shows Enforced. 

How can I fix this?

**Comment by DanGLES3** [comment]
> Im experiencing this now with PlayIntegrityFix v18.0, TB Checker "SE linux Flag is Enabled = Fail" where it always passed. Also in DevInfo "SELinux = Unable to Determine" app has root permission. Kernel SU shows enforced and command line "getenforce" shows Enforced.
> 
> How can I fix this? (Galaxy TAB S9+)

As for why it started failing when updating to 18.0, previous versions used to delete that property needlessly (even when it was set to 1) just to appease broken detection apps

**Comment by essence25** [comment]
Yes, you are correct. I just went back to v17.9 to test and TB Checker will pass again all root tests. Hmm... So the detection apps need to be updated in that case. Thank you for clarifying.

**Comment by DanGLES3** [comment]
> Yes, you are correct. I just went back to v17.9 to test and TB Checker will pass again all root tests. Hmm... So the detection apps need to be updated in that case. Thank you for clarifying.

The only checkers that have this broken check are those based off of rootbeer which is very outdated checking library that very few real app use and even when they do they ignore the broken selinux check

**Comment by essence25** [comment]
Funny thing is RootBeerFresh passes with "NOT ROOTED" on v18.0 but fails on v17.9 (SE linux Flag), so sticking with v18.0. 
All good.
